.vscode/settings.json

```diff
@@ -21,6 +21,9 @@
         "docs/doxygen": true,
         "ctx.c": true,
         "ver/current": true,
+        "ver/us/build": true,
+        "ver/jp/build": true,
```

Comment:
Might be worth adding exceptions for the map files

Reply:
I think they should be filtered out by default, it just pollutes stuff when searching in my experience